micro-rdk/src/native/entry.rs

```diff
@@ -120,6 +135,117 @@ pub async fn serve_web_inner(
     srv.serve(robot).await;
 }
 
+#[cfg(feature = "provisioning")]
+async fn serve_provisioning_async<S>(
+    ip: Ipv4Addr,
+    exec: NativeExecutor,
+    info: ProvisioningInfo,
+    storage: S,
+    last_error: Option<String>,
+) -> Result<(AppClientConfig, NativeTlsServerConfig), Box<dyn std::error::Error>>
+where
+    S: CredentialStorage + Clone + 'static,
+    S::Error: Debug,
+    GrpcError: From<S::Error>,
+{
+    let hostname = format!(
+        "provisioning-{}-{}",
```

Comment:
Should this `provisioning` string include some more detail? This is the viam v1 provisioning API that is served here.

Reply:
Maybe but not sure :)